GitUI/UserControls/FileStatusDiffCalculator.cs

```diff
@@ -15,8 +15,9 @@ public sealed partial class FileStatusDiffCalculator
         // Currently bound revisions etc. Cache so we can reload the view, if AppSettings.ShowDiffForAllParents is changed.
         private FileStatusDiffCalculatorInfo _fileStatusDiffCalculatorInfo = new();
 
-        public Func<ObjectId?, string>? DescribeRevision { get; set; }
-        public Func<GitRevision, GitRevision?>? GetActualRevision { get; set; }
+        // Default helper functions, can be set
+        public Func<ObjectId, string> DescribeRevision { get; set; } = objectId => objectId.ToShortString();
+        public Func<GitRevision, GitRevision> GetActualRevision { get; set; } = rev => rev;
```

Comment:
These are public API, and thus must be either nullable with necessary checks, or validate for null.
This goes up the stack.